.circleci/config.yml

```diff
@@ -109,15 +164,29 @@ commands:
             pip install --upgrade pip
             pip install poetry==2.0.0
 
+  upload_to_gcs:
+    parameters:
+      source:
+        type: string
+      destination:
+        type: string
+      extension:
+        type: enum
+        enum: ["xml", "json"]
+    steps:
+      - run:
+          name: Upload << parameters.source >> << parameters.extension >> Files to GCS
+          command: |
+            FILES=$(find << parameters.source >> -maxdepth 1 -type f -name "*.<< parameters.extension >>")
```

Comment:
FWIW, you could also do `$(ls -1 << parameters.source>>/*.<< parameters.extension>> )` which would do the same thing. I'm not opposed to using `find` like this, though.